modin/pandas/test/test_io.py

```diff
@@ -259,28 +259,36 @@ def _make_parquet_dir(
 @pytest.mark.filterwarnings(default_to_pandas_ignore_string)
 class TestCsv:
     # delimiter tests
-    @pytest.mark.parametrize("sep", [None, "_", ",", ".", "\n"])
-    @pytest.mark.parametrize("delimiter", ["_", ",", ".", "\n"])
```

Comment:
These parameters mean the same thing. When using them at the same time, an exception occurs - I put it in a separate test.